app/bootstrap.php

```diff
@@ -33,6 +38,21 @@ function ( Request $request, Application $app ) {
 	Application::EARLY_EVENT
 );
 
+$app->before( function( Request $request ) use ( $ffFactory ) {
+	if ( $ffFactory->getCookieHandler() === null ) {
+		$ffFactory->setCookieHandler( new BrowserCookieHandler( $request->cookies ) );
+	}
+}, Application::EARLY_EVENT );
+
+$app->after(
+	function ( Request $request, Response $response ) use ( $ffFactory ) {
+		$cookieHandler = $ffFactory->getCookieHandler();
```

Comment:
I struggled with the `after` function a bit, maybe you can add the comment that explains what is does, e.g. "Store Cookie values that were set in the route handlers"